taurus/entity/object/ingredient_run.py

```diff
@@ -12,6 +12,25 @@ class IngredientRun(BaseObject, HasQuantities):
 
     Ingredients annotate a material with information about its usage in a process.
 
+    *  If a user invokes the IngredientRun constructor in citrine-python with either the name
+       or labels argument, a deprecation warning will be issued
```

Comment:
It feels wrong to reference citrine-python behavior in taurus. Similarly, bullets 3 & 4 refer to API calls and 400 codes, neither of which are relevant in this context.